src/pages/TeamSettings/Members.vue

```diff
@@ -230,7 +235,6 @@ export default {
       pollInterval: 10000,
```

Comment:
I think the best fix is to move the where we set the role but could/should we also extend this poll interval?  Can't remember if there was a reason we need to be polling so frequently here?

Reply:
I think we had this at 60 s previously but it was leading to some confusion when joining teams/making updates.